app/views/optOut/ConfirmedOptOut.scala.html

```diff
@@ -88,6 +78,37 @@ <h2 class="govuk-heading-m">@messages("optout.confirmedOptOut.report-quarterly-n
     </ul>
 }
 
+
+@yourReportingFrequencyBlock = @{
+    val reportingFrequencyUrl = controllers.routes.ReportingFrequencyPageController.show().url
+    val reportingFrequencyLink = link(
+        id = Some("your-reporting-frequency-link"),
+        link = reportingFrequencyUrl,
+        messageKey = "optout.confirmedOptOut.yourReportingFrequency.text"
+    )
+
+    Html(messages("optout.confirmedOptOut.yourRevisedDeadlines.desc2", reportingFrequencyLink.toString()))
+}
+
+@revisedDeadlinesBlock = {
+    <div id="revised-deadlines" class="govuk-!-margin-bottom-7 govuk-!-margin-top-8">
+        <h2 id="revised-deadlines-heading" class="govuk-heading-m" >@messages("optout.confirmedOptOut.yourRevisedDeadlines.h2")</h2>
+        <p id="revised-deadlines-p1" class="govuk-body" >@messages("optout.confirmedOptOut.yourRevisedDeadlines.desc1", viewModel.startYear, viewModel.endYear, viewModel.nextYear)</p>
```

Comment:
The section of the text that says `31 January <Year>` should be bold.

Reply:
good spot, thank you 😃